node-fs-writefile/write-rand.js

```diff
@@ -0,0 +1,3 @@
+let fs = require('fs');
+
+fs.writeFile('random.txt', Math.random().toString() + '\n');
```

Comment:
When you concatenate a number to a string it will automatically be converted to a string so you don't have to use `toString()` here

Reply:
i think i had that from when i didn't add the newline concat yet thanks for the heads up!